ui/main/src/app/store/effects/card-operation.effects.ts

```diff
@@ -49,26 +51,47 @@ export class CardOperationEffects {
             map((loadedCardAction: LoadLightCard) => loadedCardAction.payload.lightCard),
             withLatestFrom(this.store.select(selectLightCardsState)),
             map(([lightCard, state]) => {
-                    if (!!lightCard.parentCardId) {
-                        return new LoadLightChildCard({lightCard: lightCard,isFromCurrentUserEntity : this.doesChildCardIsFromCurrentUserEntity(lightCard)});
-                    }
-                    else {
-                        let card: LightCard  = {...lightCard} ;
-                        card.hasChildCardFromCurrentUserEntity = this.doesNewCardVersionHasChildFromCurrentUserEntity(state.entities[card.id],card);
-                        return new LoadLightParentCard({lightCard: card});
+                if (!!lightCard.parentCardId) {
+                    const isFromCurrentUser = this.doesChildCardIsFromCurrentUserEntity(lightCard);
+                    if (isFromCurrentUser) {
+                        if (!state.entities[lightCard.parentCardId]) {
+                             // if parent does not exist yet keep in memory the information that the card with id=parentCardId has a child
+                             // and that this child if from current user entity
+                             // this situation can arise as we do not have the card ordered when received from
+                             // the back via subscription (i.e a child can be received  before his parent)
+                            this.orphanedChildCardsFromCurrentEntity.add(lightCard.parentCardId);
+                            // do not need to load this child card
+                            return new NoopAction();
```

Comment:
I don't think you need to define a dummy action, I changed the code to just "return;" and tried running the unit tests, and they passed. No complaint from ng serve either.

Reply:
it complains in the console log